src/services/ui/package.json

```diff
@@ -15,9 +15,13 @@
   },
   "dependencies": {
     "@aws-amplify/auth": "^5.4.0",
+    "@cmsgov/design-system": "2.7.1",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
     "@enterprise-cmcs/macpro-ux-lib": "1.22.1",
+    "@fortawesome/fontawesome-svg-core": "^1.2.35",
+    "@fortawesome/free-solid-svg-icons": "5.14.0",
+    "@fortawesome/react-fontawesome": "^0.1.14",
```

Comment:
i know we are using the `@heroicons/react/24/outline` package in other places, if we can replace them with their equivalent here lets do that, if they dont exist or if they dont line up its fine to use these.